### PR TITLE
Upgrade arrow to include pytorch fix

### PR DESCRIPTION
**Pull request by pcmoritz**

### CODE REVIEW

File: thirdparty/scripts/build_arrow.sh [hunk 1]
```diff
@@ -109,6 +109,22 @@ build_arrow() {
   make VERBOSE=1 -j$PARALLEL
   make install
 
+  cmake -DCMAKE_BUILD_TYPE=Release \
```
Comment (robertnishihara) [inline thirdparty/scripts/build_arrow.sh]: why run `cmake` a second time?

Comment (pcmoritz) [reply thirdparty/scripts/build_arrow.sh]: That was an accidental commit, I'm removing it

**Comment by pcmoritz** [comment]
jenkins retest this please

**Comment by pcmoritz** [comment]
jenkins retest this please

**Review by robertnishihara — APPROVED** [review]

**Comment by robertnishihara** [comment]
jenkins retest this please

**Pull request merged**